includes/definitions/discovery/zynos.yaml

```
@@ -26,6 +26,61 @@ modules:
                 oid: sysMgmtCPU5SecUsage
                 num_oid: '.1.3.6.1.4.1.890.1.15.3.2.7.{{ $index }}'
     sensors:
+        pre-cache:
+            data:
+                -
+                    oid:
+                        - ifDescr
+                        - zyStackingSlotInfoStackingStatus
```

Almost sure this one is not necessary, as it is part of zyxelStackingSlotInfoTable so it will be already loaded. 
```suggestion
                        - zyStackingSlotInfoStackingStatus
```

---

Once again forgot to remove the pre-cache after I stopped using zyStackingSlotInfoStackingStatus with different tables.